paddle/phi/infermeta/unary.cc

```diff
@@ -4228,7 +4228,9 @@ void TransposeInferMeta(const MetaTensor& x,
   int x_rank = x_dims.size();
   int axis_size = axis.size();
 
-  PADDLE_ENFORCE_EQ(
+  // Note: x_rank > axis_size when fuse squeeze2 + transpose2, else x_rank ==
+  // axis_size
```

Comment:
This note doesn't match `PADDLE_ENFORCE` error message below

Reply:
Thank you! I will update message in next PR.